pkg/controller/operators/olm/operator.go

```diff
@@ -449,6 +449,22 @@ func newOperatorWithConfig(ctx context.Context, config *operatorConfig) (*Operat
 		return nil, err
 	}
 
+	// Register CustomResourceDefinition QueueInformer (for apiextensions.k8s.io/v1 CRDs only)
+	crdV1Informer := extinf.NewSharedInformerFactory(op.opClient.ApiextensionsInterface(), config.resyncPeriod()).Apiextensions().V1().CustomResourceDefinitions()
```

Comment:
remove `v1beta1` informer in preference of `v1`

when you list v1s in the cluster, the cluster should autoconvert v1beta1 to v1 according to @ecordell